### ISSUE TITLE
Utilize the WP_List_Table class in core

### ISSUE THREAD

**Issue by frankiejarrett** [fjarrett/user-session-control#10]
As suggested by @szepeviktor in [this support forum thread](https://wordpress.org/support/topic/have-you-got-my-review?replies=2), using the `WP_List_Table` class provided by core would be a cleaner approach.

**Comment by szepeviktor** [comment]
Copy [the class](https://github.com/WordPress/WordPress/blob/master/wp-admin/includes/class-wp-list-table.php) into the plugin, rename the class and extend it. Even actions have a method in it.

**Comment by frankiejarrett** [comment]
Yes, thanks @szepeviktor. We actually [did this](https://github.com/wp-stream/stream/blob/develop/classes/class-wp-stream-list-table.php) with the WP Stream plugin list table and it works great.
